litmus-portal/frontend/src/views/CreateWorkflow/TuneWorkflow/index.tsx

```diff
@@ -254,8 +256,11 @@ const TuneWorkflow = forwardRef((_, ref) => {
     }
   }, [engineDataLoading, experimentDataLoading]);
 
-  // console.log(generatedYAML);
   function onNext() {
+    if (customWorkflow && experiment.length === 0) {
```

Comment:
=== 0 might not be needed !experiment.length can work

Reply:
It won't work without the assertion check because `if` condition works on a true or false basis but only `experiment.length` will return 0
Thus the condition becomes `if(true && 0){}`
Whereas here it states `if(true && true){}`

Just tested this locally

Reply:
@S-ayanide PTAL

> TCs:

<img width="299" alt="Screenshot 2021-04-06 at 4 41 50 PM" src="https://user-images.githubusercontent.com/50652677/113702420-1a769680-96f7-11eb-866a-e8eb1fe3e3b5.png">

> Results:

<img width="301" alt="Screenshot 2021-04-06 at 4 42 10 PM" src="https://user-images.githubusercontent.com/50652677/113702432-1d718700-96f7-11eb-8834-c72d89cc2fcf.png">

Reply:
!experiment.length will return true...but I feel since it's an if condition and multiple cases are there ===0 is more readable